pytest_rich.py

```diff
@@ -243,8 +244,44 @@ def pytest_sessionfinish(
             self.console.print(Rule("FAILURES", style="red"))
             for nodeid, report in self.failed_reports.items():
                 tb = RichExceptionChainRepr(nodeid, report.longrepr)
+                error_messages[nodeid] = tb.error_messages
                 self.console.print(tb)
 
+        if self.config.getoption("verbose") >= 0:
+            self.print_summary(error_messages, all=self.config.getoption("verbose") > 0)
+
+    def print_summary(self, error_messages, all=False):
+        total_text = Text(f" Total::{self.total_items_completed}", style="bold blue")
+        success_text = Text(
+            f"Success::{self.total_items_completed - len(self.failed_reports)}",
+            style="bold green",
+        )
+        failed_text = Text(f" Failed::{len(self.failed_reports)}", style="bold red")
+        self.console.print(
+            Rule(total_text
+                + success_text
+                + failed_text,
+                characters="=",
+                style="blue",
+            )
```

Comment:
Spacing is a bit off with the spaces in the total and failed `Text` objects:

![image](https://user-images.githubusercontent.com/19896267/198494076-0bbdf526-5aa7-4e7f-8f4f-7cba7bdbd8ef.png)

Could you change this to adding the spacing when printing to the console?

```python
total_text = Text(f"Total::{self.total_items_completed}", style="bold blue")
success_text = Text(
	f"Success::{self.total_items_completed - len(self.failed_reports)}",
    	style="bold green",
	)
failed_text = Text(f"Failed::{len(self.failed_reports)}", style="bold red")
self.console.print(
    Rule(
        total_text + " " + success_text + " " + failed_text,
        characters="=",
        style="blue",
    )
)
```

![image](https://user-images.githubusercontent.com/19896267/198494178-be845194-fa01-4fb9-93f6-b06eb444fa82.png)